### PR TITLE
ECCI-472: style restricted content notices

### PR DESCRIPTION
**Pull request by kurtisRogers-Nomensa**
- detail changes...

### CODE REVIEW

File: themes/custom/essex/templates/restricted-content/localgov-restricted-content-sign-in-with-message.html.twig [hunk 1]
```diff
@@ -1,8 +1,9 @@
 <!--cludooff: index-->
-<div>
-    <p>{{ 'This page is restricted, please sign in to see the full page.'|t }}</p>
-    <div>
-        <a href="/user/login?destination={{ path('<current>') }}">{{ 'Sign in'|t }}</a>
-    </div>
+<div class="ecc-restricted-content">
+  <p class="h3">{{ 'This area is restricted to authorised employees only. Please sign in to access this information.'|t }}</p>
+  <a class="button" href="/user/login?destination={{ path('<current>') }}">
+    {# svg #}
```
Comment (Polynya) [inline themes/custom/essex/templates/restricted-content/localgov-restricted-content-sign-in-with-message.html.twig:5]: Is this needed before the PR is ready?

Comment (kurtisRogers-Nomensa) [reply themes/custom/essex/templates/restricted-content/localgov-restricted-content-sign-in-with-message.html.twig:5]: @Polynya yeah but not too sure on the best way to add an svg in the template

Comment (kurtisRogers-Nomensa) [reply themes/custom/essex/templates/restricted-content/localgov-restricted-content-sign-in-with-message.html.twig:5]: SVG filepath
themes/custom/essex/assets/images/icons/lock.svg

Comment (kurtisRogers-Nomensa) [reply themes/custom/essex/templates/restricted-content/localgov-restricted-content-sign-in-with-message.html.twig]: Added as an inline element
image
image

**Review by Polynya — APPROVED** [review]

**Pull request merged**